client/analytics/report/products/config.js

```diff
@@ -38,10 +38,12 @@ const filterConfig = {
 		{
 			label: __( 'Single Product', 'wc-admin' ),
 			value: 'select_product',
+			chartMode: 'item-comparison',
```

Comment:
`chartMode` will need to be added to the propTypes definition in `/components/filter/index.js` for documentation:

https://github.com/woocommerce/wc-admin/blob/fe46fe1955b75fa8923af2e80d54b792c38a994b/packages/components/src/filters/filter/index.js#L237-L260

Reply:
Fixed!

https://github.com/woocommerce/wc-admin/pull/884/commits/5f81c33a3fb3e31bf59b7ab76844c767f89114c8